homework/CSharpCollections1/Program.cs

```diff
@@ -0,0 +1,143 @@
+﻿#region Task_1
+//Random random = new Random();
+//int[] array = new int[10];
+
+//Console.WriteLine("Array:");
+
+//for (int i = 0; i < array.Length; i++)
+//{
+//    array[i] = random.Next(1, 9);
+//    Console.Write("{0} ", array[i]);
+//}
+
+//Array.Sort(array);
+
+//Console.WriteLine("\nArray of even numbers");
+
+//for (int i = 0; i < array.Length; i++)
+//{
+//    if (array[i] % 2 == 0)
+//    {
+//        Console.Write("{0} ", array[i]);
+//    }
+//}
+
+//Console.WriteLine("\nArray of odd numbers");
+
+//for (int i = 0; i < array.Length; i++)
+//{
+//    if (array[i] % 2 != 0)
+//    {
+//        Console.Write("{0} ", array[i]);
+//    }
+//}
+#endregion
+#region Task_2
+//using System.Linq;
+
+//Random random = new Random();
+//int[] array = new int[10];
+
+//Console.WriteLine("Array:");
+
+//for (int i = 0; i < array.Length; i++)
+//{
+//    array[i] = random.Next(-20, 20);
+//    Console.Write("{0} ", array[i]);
+//}
+
+//Console.WriteLine();
+
+//int max = array.Max();
+//int min = array.Min();
+//int difference = max - min;
+//Console.WriteLine($"Max = {max}");
+//Console.WriteLine($"Min = {min}");
+//Console.WriteLine($"Difference = {difference}");
+#endregion
+#region Task_3
+//string[] surname = new string[] { "Peterson", "Morales", "Jones", "Morgan", "Lamb", "Collins",
+//    "Burgess", "Perez", "Ruiz", "Johnston" };
+
+//var surnameList = new List<string>();
+//surnameList.AddRange(surname);
+
+//surnameList.Sort();
+
+//Console.WriteLine("List in sorted order: ");
+//foreach (var name in surnameList)
+//    Console.Write("{0} ", name);
+
+//Console.WriteLine();
+#endregion
+#region Task_4
+//using System.Collections.Generic;
```

Comment:
You don't implement "It is required to compile a new list of voters living at a given address in alphabetical order.".
You should create new list with Citizens who live at the given address (the same address for citizens).